### ISSUE TITLE
sticky cookie domain not set by traefik v2

### ISSUE THREAD

**Issue by Distortedlogic** [traefik/traefik#7463]
Do you want to request a *feature* or report a *bug*?

feature request

When using sticky sessions with traefik, setting the cookie name works via traefik, but traefik does not have an option to set the cookie domain

### What did you do?
docker swarm stack compose file
```
services:
  traefik:
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./traefik/traefik.prod.yml:/etc/traefik/traefik.yml
      - acme:/etc/traefik/acme
    networks:
      - web
    ports:
      - 80:80
      - 443:443
    deploy:
      mode: global
      update_config:
        parallelism: 1
        delay: 10s
      labels:
        - traefik.enable=false
      restart_policy:
        condition: on-failure
  backend:
    networks:
      - web
      - backend
    deploy:
      labels:
        - traefik.enable=true
        - traefik.http.routers.backend.rule=Host(`backend.memehub.lol`)
        - traefik.http.routers.backend.entrypoints=secure
        - traefik.http.services.backend.loadbalancer.server.port=5000
        - traefik.http.routers.backend.tls.certresolver=letsencrypt
        - traefik.http.services.backend.loadbalancer.sticky.cookie=true
        - traefik.http.services.backend.loadbalancer.sticky.cookie.name=sticky_speed_cow
        - traefik.http.services.backend.loadbalancer.sticky.cookie.httpOnly=true
        - traefik.http.services.backend.loadbalancer.sticky.cookie.domain=".memehub.lol"
        - traefik.http.services.backend.loadbalancer.sticky.cookie.secure=true
      replicas: 1
      update_config:
        parallelism: 2
        delay: 10s
      restart_policy:
        condition: on-failure
```
traefik.yml

```
log:
  level: DEBUG
  filepath: /etc/traefik/log/traefik.log
api:
  dashboard: true
  insecure: true  
  debug: true
entryPoints:
  unsecure:
    address: :80
  secure:
    address: :443
providers:
  docker:
    exposedByDefault: false
    network: web
    swarmMode: true
certificatesResolvers:
  letsencrypt:
    acme:
      email: jermeek@gmail.com
      storage: /etc/traefik/acme/acme.json
      tlsChallenge: {}
      httpChallenge:
        entryPoint: unsecure
```

### What did you expect to see?

A cookie with domain specified by the dev in labels or yml
ie cookie domain of ".memehub.lol" that is httpOnly and secure in my case

### What did you see instead?

Cookie domain self set by traefik.

![26-10-2020-07-16-11](https://user-images.githubusercontent.com/54854447/97166070-064b6800-175b-11eb-9bb1-bba4a9bb7efb.png)

ACTUALLY if you try to use
```-traefik.http.services.backend.loadbalancer.sticky.cookie.domain=".memehub.lol"```
then traefik breaks and throws CORS errors.

### Why is this a problem?

traefik sits between my frontend and backend. So the cookie domain set by traefik does not match the domain of the frontend

ie fontend domain - "memehub.lol". and backend domain - "backend.memehub.lol"
Traefik sets the cookie domain to "backend.memehub.lol" as illustrated in the screenshot above.

When the frontend is refreshed, if the cookie domain is not matching the current domain/subdomain then the cookie is lost on the refresh.

After traefik sets the initial sticky cookie, if I manually change the domain of the cookie to the desired cookie domain ".memehub.lol" then the cookie persists through refreshes without issue, BUT, if the cookie domain is left as traefik self sets it "backend.memehub.lol" then the cookie is lost on refresh.

When the cookie is lost, then when replicas for the backend service is more than one, the requests dont get routed to the same server each time as desired, but get defaulted to round robin

**Comment by rtribotte** [comment]
Hello @Distortedlogic,

Thanks for your contribution !

Can you edit the issue to be a feature request, to be more consistent :-)

**Comment by Distortedlogic** [comment]
@rtribotte 

I added a PR to fix this

https://github.com/traefik/traefik/pull/7473

**Comment by rtribotte** [comment]
Hello @Distortedlogic,

As @kevinpollet answered you on the pull request, we tried to reproduce the issue without success.
We indeed tried an ajax request on a sub domain different from the "frontend" page (with http and also https).
With CORS configuration and a request `withCredentials` to true, we see the cookie being reused by the user agent which send it back to the "backend".

We believe that making the domain configurable is not necessary regarding your use case, but we are looking forward for another use case that would make it obvious.

As we overlooked this issue to be an enhancement, I'll label this issue as a bug possible, but keep the discussion open.
I invite you to share with us a reproducible case so that we can see the problem.

I hope this is more comprehensible.
Thank you for your time and your interest in Traefik
